TestObjectiveDropbox/TestObjectiveDropbox_iOSTests/TestAsciiEncoding.m

```diff
@@ -41,14 +29,40 @@ - (void)tearDown {
         @"this has a clustered flag 🇺🇸": @"this has a clustered flag \\ud83c\\uddfa\\ud83c\\uddf8",
         @"this is a big emoji 👩‍👩‍👧‍👦": @"this is a big emoji \\ud83d\\udc69\\u200d\\ud83d\\udc69\\u200d\\ud83d\\udc67\\u200d\\ud83d\\udc66",
         @"🍺": @"\\ud83c\\udf7a",
-        @"this\nhas some whitespace": @"this\nhas some whitespace"
+        @"this\nhas some whitespace": @"this\nhas some whitespace",
+        @"héllø wörld": @"h\\u00e9ll\\u00f8 w\\u00f6rld"
+    };
+}
+
+- (void)testEveryUnichar {
```

Comment:
Probably doesn't matter but there are more unichars than this aren't there? http://www.unicode.org/faq/utf_bom.html#gen0